htlcswitch/link.go

```diff
@@ -1193,6 +1210,27 @@ func (l *channelLink) htlcManager() {
 				"PendingLocalUpdateCount")
 		}
 
+		format, err := l.trySendingShutdown()
+		if err != nil {
+			l.fail(LinkFailureError{
+				code: ErrInternalError,
+			}, format, err)
```

Comment:
This format/error is just used by `l.fail` to log the error down the line. Seems more typical to just return an error from `trySendingShutdown` (can wrap the error if desired) then add some generic formatting string here if we want it.